### DISCUSSION TITLE
Having this issue

### DISCUSSION THREAD

**Discussion by dipto0321** [OpenBangla/OpenBangla-Keyboard#216] [Q&A] [answered]
Hi, for the last couple of days I'm having trouble with this
![image](https://user-images.githubusercontent.com/6507903/117416550-4d50bc00-af3b-11eb-97b9-8ebe683843a4.png)

How can I get rid of this? 

Thanks

**Comment by dipto0321** [comment, marked as answer]
Figure out the solution. Need to remove openbangle related stuff from `pacman.conf` to get rid of this kind of issue.

**Reply by mominul** [reply]
Yeah, we have already published [a blog post](https://openbangla.github.io/blog/bintray-notice/) about this matter.

**Comment by dipto0321** [comment]
Thanks for the update.




On Fri, May 7, 2021 at 6:25 PM Muhammad Mominul Huque <
***@***.***> wrote:

> Yeah, we have already published a blog post
> <https://openbangla.github.io/blog/bintray-notice/> about this matter.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenBangla/OpenBangla-Keyboard/discussions/216#discussioncomment-709036>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABRU274EC6AUVLTDABMHVHTTMPL2PANCNFSM44JHD3HA>
> .
>


-- 

Dipto Karmakar
Full-Stack Developer
WhatsApp/t.gram:  +8801732646382
site:  diptokarmakar.me
email:  ***@***.***
skype:  dipto0321
<https://www.linkedin.com/in/diptokarmakar/>
<https://twitter.com/Diptokmk47>
<https://angel.co/dipto-karmakar>
<https://github.com/dipto0321>
<https://gitlab.com/dipto0321>

**Reply by ahmubashshir** [reply]
you can configure chaotic-aur repo :3
https://aur.chaotic.cx
